flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowingTestHarnessTest.java

```diff
@@ -84,6 +86,84 @@ public void testEventTimeTumblingWindows() throws Exception {
 	}
 
 	@Test
+	public void testEventTimeTumblingWindowsWithOffset() throws Exception {
```

Comment:
These tests should go into `WindowOperatorTest`. The existing tests here are just meant to exercise the `WindowTestHarness`.